scalafix-core/shared/src/main/scala/scalafix/internal/util/ClassloadRule.scala

```diff
@@ -115,7 +115,7 @@ class ClassloadRule[T](classLoader: ClassLoader)(implicit ev: ClassTag[T]) {
 }
 
 object ClassloadRule {
-  lazy val defaultClassloader = getClass.getClassLoader
+  def defaultClassloader: ClassLoader = getClass.getClassLoader
```

Comment:
Why is this no longer lazy?

Reply:
not necessary change, just remove lazy because lazy should only be used when necessary.